src/main/resources/templates/header.html

```diff
@@ -2,10 +2,15 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org"
       xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
 <head>
-    <link href="/css/style.css" rel="stylesheet"/>
-    <script src="/js/jquery.min.js"></script>
-    <script src="/js/d3.min.js"></script>
-    <script src="/js/config.js"></script>
+    <!--<link href="/css/style.css" rel="stylesheet"/>-->
```

Comment:
Old version of the code is stored in git, both locally and on GitHub - it will not be lost. 
If you want to retrieve old version of the code - you just need to do 
 
    git checkout <commit-hash>

where commit hash can be found via 
 
    git log

or you can find it on GitHub. There is no need to keep this code commented out.